activities/loadtest/loadtest.go

```diff
@@ -167,12 +168,34 @@ func (a *Activity) RunLoadTest(ctx context.Context, chainState []byte,
 
 	chain, err := chain.RestoreChain(ctx, logger, p, chainState, node.RestoreNode, testnet.CosmosWalletConfig)
 	if err != nil {
-		return PackagedState{}, err
+		newProviderState, serializeErr := p.SerializeProvider(ctx)
```

Comment:
should we extract this out into a function? where we serialize as much as possible and just return it